packages/codegen-ui-react/lib/forms/form-renderer-helper.ts

```diff
@@ -280,9 +284,159 @@ export const buildFormPropNode = (form: StudioForm) => {
   return factory.createTypeLiteralNode(propSignatures);
 };
 
+export const buildStateMutationStatement = (name: string, defaultValue: any) => {
```

Comment:
can we type this to an expression?
```suggestion
export const buildStateMutationStatement = (name: string, defaultValue: Expression) => {
```

Reply:
Yup. Will update